.github/workflows/cypress_eyeson.yml

```diff
@@ -0,0 +1,42 @@
+name: Run Cypress EyesOn Tests
+
+on:
+  workflow_dispatch
```

Comment:
Perfect, I like this setup. The test exists and we can run it when we choose to. =)